README.adoc

```diff
@@ -132,17 +123,6 @@ is to install following packages:
 - ovirt-engine-wildfly
 - ovirt-engine-wildfly-overlay
 
-Both packages can be installed from ovirt-master-snapshot-static repository:
-
-  [ovirt-master-snapshot-static]
-  name=Latest oVirt master additional nightly snapshot
-  baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot-static/rpm/@DIST@$releasever/
-  enabled=1
-  skip_if_unavailable=1
-  gpgcheck=0
-
-Please replace `@DIST@` with `fc` for Fedora or `el` for Centos/RHEL.
-
 Alternatively, repository list can be updated.
```

Comment:
This section comes as an alternative to the `ovirt-master-snapshot-static` repository that you deleted and both were related to `WildFly`'s 2 packages  installation.
IMHO lines 126-128 (new version) should become:
```
Both packages can be installed from CentOS Stream 8 repositories.
Repository list can be updated using the following commands:
```